### ISSUE TITLE
Fire Aspect & 1.21

### ISSUE THREAD

**Issue by Tomographix** [ploppyperson/StackMob-5#292]
Hi, The Fire Aspect doesn't seem to work on stacked mobs in 1.21.

Running:
- PaperSpigot 1.21 (Latest build as of 15 July
- StackMob 5.9.1

**Comment by ploppyperson** [comment]
What exactly isn't working?

**Comment by Tomographix** [comment]
Fire Aspect is a vanilla Minecraft sword enchantment that sets the target on fire. When a mob that normally drops raw meat is killed while it is burning, or if a mob is killed in one shot, even if it is in water, it drops cooked meat.

In this situation, if you kill a stack of 200 pigs & 90% of the drops are raw porkchops when they should be 100% cooked porkchop - or at least this is how it worked prior to 1.21.

**Comment by ploppyperson** [comment]
I can only reproduce this when a mob is in rain. Is this the same for you?

**Comment by Tomographix** [comment]
Ah no, rain is actually disabled on the server. Have got a short video here on Imgur: https://i.imgur.com/EdnbjZF.mp4

**Comment by ploppyperson** [comment]
Yeah so after some more testing seems to be that if you kill the mob in one hit then fire aspect doesn't apply, but if there are two hits it works fine... This may be a craftbukkit bug but needs further looking into.

Temp fix would be to disable use of loot tables in config.

**Comment by ploppyperson** [comment]
Thanks for the error report.
Fixed in cde56de
